libkbfs/prefetcher.go

```diff
@@ -135,12 +164,18 @@ func (p *blockPrefetcher) prefetchDirectDirBlock(b *DirBlock, kmd KeyMetadata, p
 		case Exec:
 			block = &FileBlock{}
 		default:
+			p.log.CDebugf(context.Background(), "Skipping prefetch for entry of unknown type %T.", entry)
```

Comment:
Same here with punctuation.  Also it would probably be useful to print `entry.BlockPointer` here.  (Name would be even better, but I don't think you have it easily available after the sort.)